tests/unit_tests/pyanaconda_tests/test_product.py

```diff
@@ -278,32 +278,6 @@ def test_rhel_products(self):
             ENTERPRISE_PARTITIONING
         )
 
-    def test_product_module_list_difference_fedora_rhel(self):
-        """Test for expected Fedora & RHEL module list differences."""
-        fedora_config = self._get_config("Fedora")
-        fedora_modules = fedora_config.anaconda.kickstart_modules
-
-        rhel_config = self._get_config("Red Hat Enterprise Linux")
-        rhel_modules = rhel_config.anaconda.kickstart_modules
-
-        difference = list(set(rhel_modules) - set(fedora_modules))
-        expected_difference = ["org.fedoraproject.Anaconda.Modules.Subscription"]
-
-        self.assertListEqual(difference, expected_difference)
-
-    def test_product_module_difference_centos_rhel(self):
-        """Test for expected CentOS & RHEL module list differences."""
-        centos_config = self._get_config("CentOS Stream")
-        centos_modules = centos_config.anaconda.kickstart_modules
-
-        rhel_config = self._get_config("Red Hat Enterprise Linux")
-        rhel_modules = rhel_config.anaconda.kickstart_modules
-
-        difference = list(set(rhel_modules) - set(centos_modules))
-        expected_difference = ["org.fedoraproject.Anaconda.Modules.Subscription"]
-
-        self.assertListEqual(difference, expected_difference)
```

Comment:
I don't think we need them since `kickstart_modules` is no longer used in these configurations, so we don't have to check the difference of these lists.

Reply:
OK, thanks for explanation.